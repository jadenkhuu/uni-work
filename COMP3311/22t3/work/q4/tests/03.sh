psql racing -c "select * from q4('hoof') order by horse;"
