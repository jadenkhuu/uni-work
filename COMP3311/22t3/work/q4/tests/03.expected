       horse       | average 
-------------------+---------
 David Hasselhoofs |  129470
 Hoof Hearted      |   13333
 Hoofy Heart       |     200
(3 rows)
